django-stubs/shortcuts.pyi

```diff
@@ -13,15 +13,15 @@ from django.template.context import Context
 
 def render_to_response(
     template_name: Union[str, Sequence[str]],
-    context: Optional[Union[Context, Dict[str, Any]]] = ...,
+    context: Optional[Union[Context, Mapping]] = ...,
```

Comment:
`Mapping` is generic. So, I guess it has to be `Mapping[str, Any]`

Reply:
sure thing